packages/pluggableWidgets/gallery-web/src/components/Gallery.tsx

```diff
@@ -10,6 +10,7 @@ export interface GalleryProps<T extends ObjectItem> {
     emptyMessageTitle?: string;
     filters?: ReactNode;
     filtersTitle?: string;
+    filtersCount: number;
```

Comment:
As I see there is no use case of this prop except checking if it is `falsy`. I think it is better to introduce a boolean prop instead.